modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -59,9 +57,9 @@ For more information about which permissions to use, see xref:api-manager::org-c
 * Obtain the Cluster Administrator role to access the Kubernetes cluster (only required for installation process).
 * If you have configured an external identity provider for your organization, create a xref:obtain-connected-apps-credentials.adoc[Connected App] in Anypoint Platform with Exchange Contributors and Manage APIs Configuration roles. Anypoint Service Mesh does not support users residing on an external identity provider for tasks, such as creating and discovering APIs.
 
-== Ports, IPs, and Hostnames Whitelist Requirements
+== Ports, IPs, and Hostnames Allow list Requirements
 
-In your network, you might need to whitelist the hostnames and ports of external resources to allow Anypoint Service Mesh to communicate with the MuleSoft-managed online Anypoint Platform APIs and services.
+You might need to allow a list of hostnames and ports of external resources in your network. This helps you to enable Anypoint Service Mesh to communicate with the MuleSoft-managed online Anypoint Platform APIs and services:
```

Comment:
I might need to allow a list of hostnames and ports in my network?
Or do you mean that I might need to allow some servers to access various ports in my network, and that I can create a list of those servers and ports?
"This helps you to enable"? How about "Doing so enables"?

Reply:
Done! Thanks!